config/redirects

```diff
@@ -1,43 +1,17 @@
 define: base https://docs.mongodb.com/bi-connector
-define: versions v1.1 v2.0 v2.1 v2.2 v2.3 v2.4 v2.5 v2.6 v2.7 v2.8 v2.9 v2.10 v2.11 v2.12 v2.13 v2.14 master
-symlink: v2.14 -> master
-symlink: upcoming -> master
-symlink: current -> master
+define: versions v2.10 v2.11 v2.12 v2.13 v2.14 master
 
-(v1.1-*]: bi-connector/${version}/reference/mongobiuser -> ${base}/${version}/components
-(v1.1-*]: bi-connector/${version}/reference/mongobischema -> ${base}/${version}/components
-[*-v1.1]: bi-connector/${version}/reference/mongosqld -> ${base}/${version}/components
-raw: bi-connector/ -> ${base}/current
-[*-v1.1]: bi-connector/${version}/tutorial/connecting -> ${base}/${version}/
-raw: bi-connector/schema-configuration -> ${base}/master/schema-configuration
-raw: bi-connector/installation -> ${base}/master/installation
-raw: bi-connector/components -> ${base}/master/components
-raw: bi-connector/components/mongobiuser -> ${base}/master/components
-raw: bi-connector/components/mongodrdl -> ${base}/master/components/mongodrdl
-raw: bi-connector/components/mongobischema -> ${base}/master/components
-raw: bi-connector/faq -> ${base}/master/faq
-raw: bi-connector/release-notes -> ${base}/master/release-notes
-[*-v2.1]: bi-connector/${version}/connect/mysql -> ${base}/${version}/tutorial/connecting
-[*-v2.1]: bi-connector/${version}/connect/tableau -> ${base}/${version}/tutorial/connecting
-(v2.1-*]: bi-connector/${version}/tutorial/connecting -> ${base}/${version}/tutorial/using-the-bi
-(v2.1-*]: bi-connector/${version}/tutorial/connecting-to-atlas -> ${base}/${version}/reference/mongosqld
-[*-v2.1]: bi-connector/${version}/atlas-bi-connector/ -> ${base}/${version}
-[*-v2.1]: bi-connector/${version}/client-dependencies/ -> ${base}/${version}
-[*-v2.1]: bi-connector/${version}/tutorial/install-bi-connector-rhel/ -> ${base}/${version}/installation
-[*-v2.1]: bi-connector/${version}/tutorial/install-bi-connector-debian/ -> ${base}/${version}/installation
-[*-v2.1]: bi-connector/${version}/tutorial/install-bi-connector-macos/ -> ${base}/${version}/installation
-[*-v2.1]: bi-connector/${version}/tutorial/install-bi-connector-windows/ -> ${base}/${version}/installation
-[v2.0-v2.1]: bi-connector/${version}/client-applications/ -> ${base}/${version}/tutorial/using-the-bi
-(v2.1-*]: bi-connector/${version}/tutorial/using-the-bi/ -> ${base}/${version}/client-applications
```

Comment:
I don't see this redirect in the added ones. Should it be?

Reply:
Discussed via Slack. This redirect still needs to be added back.

Reply:
Added back in on line 39.